torch2trt/test.py

```diff
@@ -20,7 +20,7 @@ def run(self):
         inputs_conversion += (torch.zeros(shape).to(self.device).type(self.dtype), )
         
     # convert module
-    module_trt = torch2trt(module, inputs_conversion, **self.torch2trt_kwargs)
+    module_trt = torch2trt(module, inputs_conversion, max_workspace_size=1 << 20,  **self.torch2trt_kwargs)
```

Comment:
max_workspace_size can be provided as argument to torch2trt_kwargs where relevant when using ``@add_module_test``